main.go

```diff
@@ -365,6 +365,19 @@ func getClosedMilestones(baseURL string, token string, projectID string, milesto
 	return editMilestones, nil
 }
 
+func validateBaseURLScheme(baseURL string) (string, error) {
+	u, err := url.Parse(baseURL)
+	if err != nil {
+		logger.Fatal(err)
```

Comment:
You are logging the error in the function, but also provide a return value for error. In the current state it will never return an error only nil. I would assume that instead of logging it in here you use `return nil, err` as you are already logging in the main func.